apps/barista/src/components/headline-link/headline-link.ts

```diff
@@ -0,0 +1,77 @@
+/**
+ * @license
+ * Copyright 2019 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, Input } from '@angular/core';
+import { DtToastRef } from 'components/toast/src/toast-ref';
+import { DtToast } from 'components/toast/src/toast';
+
+@Component({
+  selector: 'ba-headline-link',
+  templateUrl: 'headline-link.html',
+  // styleUrls: ['./icon-color-wheel.scss'],
+})
+export class BaHeadlineLink {
+  @Input() id: string;
+
+  /** Message which is displayed when the link to the headline is successfully copied */
+  private _toastMessage = 'Copied to clipboard';
```

Comment:
Do we need a variable here or can we just add the string in the function where it's used?